packages/ag-charts-community/src/chart/mapping/prepareSeries.ts

```diff
@@ -89,6 +101,13 @@ export function processSeriesOptions(_opts: AgChartOptions, seriesOptions: Serie
             Logger.warnOnce(`unsupported stacking of series type: ${sType}`);
         }
 
+        // AG-10089 Ignore duplicate user input id values:
+        if (idMatchIndex !== undefined) {
+            Logger.warn(`series[${index}].id "${series.id}" ignored because it duplicates series[${idMatchIndex}].id`);
```

Comment:
Nit: This should be warnOne() to avoid spamming the console.

Reply:
Done.